paasta_tools/cli/cmds/logs.py

```diff
@@ -167,7 +168,24 @@ def prefix(input_string, component):
     return "%s: %s" % (LOG_COMPONENTS[component]['color'](component), input_string)
 
 
-def paasta_log_line_passes_filter(line, levels, service, components, clusters):
+def check_timestamp_in_range(timestamp, start_time, end_time):
+    """A convenience function to check if a timestamp is within the given start and end times,
+    returns true if start_time or end_time is None
```

Comment:
I don't know if this really makes sense for the API you have, but maybe if start_time is None you could treat it as `float('-Infinity')` and if end_time is None you could treat it as `float('Infinity')`. If both were undefined, you'd have the same behavior - all timestamps would pass the check; but if e.g. only end_time were undefined you'd have a more sensible behavior: anything after end_time would be filtered out, anything before it would pass.